library/src/main/java/com/cunoraz/tagview/TagView.java

```diff
@@ -316,13 +310,6 @@ public void addTags(String[] tags) {
             addTag(tag);
         }
     }
-	/*public void addTags(ArrayList<String> tags){
```

Comment:
I commented this method because I want people can send String Arraylist for tags.